app/libs/js/LEAF/formQuery.js

```diff
@@ -222,6 +231,13 @@ var LeafFormQuery = function () { //NOTE: keeping this a var in case custom code
     abortSignal = signal;
   }
 
+  /**
+   * encodeReadableURI provides minimal character URI encoding, prioritizing readible URLs
+   */
+  function encodeReadableURI(url) {
```

Comment:
Should this be more future-proofed?  The name gives the impression that it does more than one thing.  So it probably should be set up initially to do so in order to ensure mistakes aren't made down the road.  Specifically, assigning the replacement to a local variable and then returning that.  That way when someone comes in to add to the functionality, they follow a pattern already established.

Reply:
I went ahead and updated it. Generally this should be avoided in this specific type of PR where we're simply re-syncing files instead of making any real changes. I've expanded the potential impact and testing to cover this new change.